WINCOMPILE.md

```diff
@@ -0,0 +1,155 @@
+    │       ├───hwloc
+    │       └───man
+    │           ├───man1
+    │           ├───man3
+    │           └───man7
+    └───libmicrohttpd-0.9.55-w32-bin
+        ├───x86
+        │   ├───MinGW
+        │   │   ├───shared
+        │   │   │   └───mingw32
+        │   │   │       ├───bin
+        │   │   │       ├───include
+        │   │   │       └───lib
+        │   │   │           └───pkgconfig
+        │   │   ├───shared-xp
+        │   │   │   └───mingw32
+        │   │   │       ├───bin
+        │   │   │       ├───include
+        │   │   │       └───lib
+        │   │   │           └───pkgconfig
+        │   │   ├───static
+        │   │   │   └───mingw32
+        │   │   │       ├───include
+        │   │   │       └───lib
+        │   │   │           └───pkgconfig
+        │   │   └───static-xp
+        │   │       └───mingw32
+        │   │           ├───include
+        │   │           └───lib
+        │   │               └───pkgconfig
+        │   ├───VS2013
+        │   │   ├───Release-dll
+        │   │   ├───Release-dll-xp
+        │   │   ├───Release-static
+        │   │   └───Release-static-xp
+        │   ├───VS2015
+        │   │   ├───Debug-dll
+        │   │   ├───Debug-dll-xp
+        │   │   ├───Debug-static
+        │   │   ├───Debug-static-xp
+        │   │   ├───Release-dll
+        │   │   ├───Release-dll-xp
+        │   │   ├───Release-static
+        │   │   └───Release-static-xp
+        │   └───VS2017
+        │       ├───Debug-dll
+        │       ├───Debug-static
+        │       ├───Release-dll
+        │       └───Release-static
+        └───x86_64
+            ├───MinGW
+            │   ├───shared
+            │   │   └───mingw64
+            │   │       ├───bin
+            │   │       ├───include
+            │   │       └───lib
+            │   │           └───pkgconfig
+            │   └───static
+            │       └───mingw64
+            │           ├───include
+            │           └───lib
+            │               └───pkgconfig
+            ├───VS2013
+            │   ├───Release-dll
+            │   └───Release-static
+            ├───VS2015
+            │   ├───Debug-dll
+            │   ├───Debug-static
+            │   ├───Release-dll
+            │   └───Release-static
+            └───VS2017
+                ├───Debug-dll
+                ├───Debug-static
+                ├───Release-dll
+                └───Release-static
+  ```
+
+## Compile
+
+- download and unzip `xmr-stak-cpu`
+- open a command line `cmd`
+- `cd` to your unzipped source code directory
+- execute the following commands (NOTE: path to VS2017 can be different)
+  ```
+  "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\Tools\VsMSBuildCmd.bat"
+  set CMAKE_PREFIX_PATH=C:\xmr-stak-dep\hwloc-win64-build-1.11.7;C:\xmr-stak-dep\libmicrohttpd-0.9.55-w32-bin\x86_64\VS2017\Release-static
+  mkdir build
+  cd build
+  cmake -G "Visual Studio 15 2017 Win64" -T v141,host=x64 ..
```

Comment:
I was unable to get this to work without extra `-DOpenSSL_ENABLE=OFF`. I don't think it is implemented in our cmake script so this should be added to the guide until finding SSL is supported too